compiler/optimizer/OMRSimplifierHandlers.cpp

```diff
@@ -11997,6 +11998,216 @@ TR::Node *sxorSimplifier(TR::Node * node, TR::Block * block, TR::Simplifier * s)
    return node;
    }
 
+template <typename T>
```

Comment:
Can you add a high level description for this function in Comment? Also same for compressBits simplifier.